### DISCUSSION TITLE
Help me to complete this task

### DISCUSSION THREAD

**Discussion by sivanaikk** [jax-ml/jax#10214] [General]
1. Animate bivariate normal distribution. 
![Screenshot_2022-04-09-22-50-27-44_a1b1bbe5f63d5b96c1a0f87c197ebfae](https://user-images.githubusercontent.com/60335740/162584682-3e30a4a0-f29a-4fe4-a540-3ae91b865a58.jpg)

Reproduce the above figure showing samples from bivariate normal with marginal PDFs from scratch using JAX and matplotlib.
Add interactivity to the figure by adding sliders with ipywidgets. You should be able to vary the parameters of bivariate normal distribution (mean and covariance matrix) using ipywidgets.

**Comment by jakevdp** [comment]
Sounds like a homework problem 😀 

The JAX function you might be looking for is [`jax.random.multivariate_normal`](https://jax.readthedocs.io/en/latest/_autosummary/jax.random.multivariate_normal.html)

I'd inquire elsewhere regarding matplotlib, ipywidgets, etc. A pro tip: if you want your question answered, it would be better to ask about a specific thing and show what you've already tried. A nice resource on asking good technical questions is StackOverflow's [How do I ask a good question](https://stackoverflow.com/help/how-to-ask). Best of luck!

**Reply by arnavraj9577** [reply]
The main problem is to create graph with help of matplotlib!
